### PR TITLE
FIX - fetch fonts task

### PR DESCRIPTION
**Pull request by killanaca**
- add fontFamily property to fonts collection inside the fetch-fonts task to fix an issue with bold and italic variants of the same font family
- add TimesModern-Regular font 
- add TimesDigitalW04-Regular font
- add TimesDigitalW04-Italic font 
- add TimesDigitalW04-Bold font

### CODE REVIEW

**Comment by killanaca** [comment]
@sericaia fonts added. thanks for the catch

File: packages/author-head/author-head.js [hunk 1]
```diff
@@ -47,7 +50,8 @@ const styles = StyleSheet.create({
     textDecorationLine: "none"
   },
   bio: {
-    fontFamily: "TimesDigital-Regular",
+    fontFamily:
+      Platform.OS === "android" ? fontFamilyAndroid : fontFamilyWebAndIos,
```
Comment (sericaia) [inline packages/author-head/author-head.js]: I would prefer to have Platform.select

File: .storybook/preview-head.html [hunk 2]
```diff
@@ -26,5 +33,26 @@
         font-weight: 400;
```
Comment (sericaia) [inline .storybook/preview-head.html]: shouldn't this be removed too?

Comment (nehasri89) [reply .storybook/preview-head.html]: just noticed this is diff than 'TimesDigital-Regular', it is 'TimesDigital-RegularSC'

**Comment by killanaca** [comment]
#227

**Pull request closed without merging**